_partials/_migrate_live_migration_docker_subcommand.md

```diff
@@ -53,7 +53,18 @@ docker run --rm -it --name live-migration-snapshot \
     -e PGCOPYDB_TARGET_PGURI=$TARGET \
     --pid=host \
     -v ~/live-migration:/opt/timescale/ts_cdc \
-    timescale/live-migration:v0.0.9 snapshot
+    timescale/live-migration:v0.0.10 snapshot
+```
+
+In addition to creating a snapshot, this process also validates prerequisites on the source and target to ensure the database instances are ready for replication.
+
+For example, it checks if all tables on the source have either a PRIMARY KEY or REPLICA IDENTITY set. If not, it displays a warning message listing the tables without REPLICA IDENTITY and waits for user confirmation before proceeding with the snapshot creation.
+
+```sh
+2024-03-25T12:40:40.884 WARNING: The following tables in the Source DB have neither a primary key nor a REPLICA IDENTITY (FULL/INDEX)
+2024-03-25T12:40:40.884 WARNING: UPDATE and DELETE statements on these tables will not be replicated to the Target DB
+2024-03-25T12:40:40.884 WARNING:        - public.metrics
+Press 'c' and ENTER to continue
```

Comment:
Looking at this again, we don't provide something actionable for the user, and IMO it's not crystal-clear that if they continue without changing the configuration in the source, UPDATE and DELETE statements will not be replicated. WDYT? At the least, I think we could point this out in the docs.

Reply:
👍